examples/dockerfile-procfile/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM gliderlabs/alpine:3.1
```

Comment:
@matthewford that's probably a good idea for another PR, and all of the other examples Dockerfiles could be updated as well. AFAIK, images are never built with these Dockerfiles - they're only used when testing various `convox init` scenarios.